etc/sample-input/copy-ftp.json

```diff
@@ -9,6 +9,8 @@
       {
         "Type": "Copy",
         "Mode": "FTP",
+        "Passive": "true",
+        "MD5": "true",
```

Comment:
These let me control how the task operates, and are things I can currently set in exchange -> fixer

Reply:
no, just boolean. fixed.